stl/src/ppltasks.cpp

```diff
@@ -19,7 +18,8 @@
 #include <ctxtcall.h>
 #include <mutex>
 #include <windows.foundation.diagnostics.h>
-#endif
+
+#pragma comment(lib, "ole32")

```

Comment:
this won't work. ppltasks is built into the "normal" dll, and we may get loaded into a process that does not have this dependency. We must use LoadLibrary/GetModuleHandle from a new satellite dll

Reply:
Note also that we can't make any existing dlls depend on the new satellite dll, it must be optional

Reply:
again, the Windows Loader takes care of loading DLL dependencies - an EXE does not have to share them